### PR TITLE
Lighthouse: Update WebP audit documentation

### PR DESCRIPTION
**Pull request by patrickhulce**
What's changed, or what was fixed?
- Updates WebP audit documentation to be more inclusive and reference other browser next-gen formats.
- Adds a matrix of browser support for each format.

**Target Live Date:** 2017-12-15 (anytime)

- [ ] This has been reviewed and approved by (@kaycebasques)
- [x] I have run `gulp test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [ ] I've staged the site and manually verified that my content displays correctly.

**R:** @kaycebasques 

Questions:
- Not sure if there's a policy on browser order so just did alphabetical
- [This](http://www.useragentman.com/blog/2015/01/14/using-webp-jpeg2000-jpegxr-apng-now-with-picturefill-and-modernizr/) was a great external blog post and using all of these formats with feature detection but not sure if we have something better or want to reference how-to on other formats

### CODE REVIEW

**Comment by petele** [comment]
@kaycebasques - once you've reviewed, feel free to merge!

**Comment by petele** [comment]
@kaycebasques - is this ready?

**Comment by kaycebasques** [comment]
On it

File: src/content/en/tools/lighthouse/audits/webp.md [hunk 1]
```diff
@@ -22,15 +25,25 @@ Click **View Details** to see the potential savings of encoding your page's
...
-support for WebP?][fallback] for an overview of fallback techniques.
+Browser support is not universal for WebP, but similar savings should be available in
+most major browsers in an alternative next-gen format. You'll need to serve a fallback PNG
+or JPEG image for other browser support. See [How can I detect browser
+support for WebP?][fallback] for an overview of fallback techniques and the table below for browser 
+support of image formats.
 
 [fallback]: /speed/webp/faq#how_can_i_detect_browser_support_for_webp
 
+| Browser | WebP | JPEG 2000 | JPEG XR | JPEG | PNG |
```
Comment (kaycebasques) [inline src/content/en/tools/lighthouse/audits/webp.md]: It'd be better to link this to somewhere else... we're not gonna remember to maintain this

**Pull request merged**